client/components/chart/style.scss

```
@@ -434,18 +434,11 @@ $y-axis-padding: 0 20px 0 10px;
 			min-width: 0;
 
 			&::before {
-				content: '';
+				@include long-content-fade( $color: var( --color-neutral-600-rgb ) );
```

Seems that after https://github.com/Automattic/wp-calypso/pull/35949 the var name has changed:

```suggestion
				@include long-content-fade( $color: var( --color-neutral-60-rgb ) );
```